demos/prefsFramework/index.html

```diff
@@ -90,7 +90,7 @@
             });
         </script>
 
-        <div class="flc-overviewPanel fl-overviewPanel-container"></div>
+        <div class="flc-overviewPanel flc-toc-exclude fl-overviewPanel-container"></div>
```

Comment:
I think this style of configuration may have been implemented backwards. Rather than requiring to intrude on all the markup in the page with a "flc-toc-exclude" class, I think it would make more sense for this to be the integrator's job - there should be a cleaner way, for example, of specifying that flc-overviewPanel, the existing selector, represents a thing should should be excluded. This requires "unbundling" the giant selector that TOC uses for exclusions into a form which is easier to configure piece by piece